src/mon/LogMonitor.cc

```diff
@@ -355,16 +354,28 @@ void LogMonitor::log_external(const LogEntry& le)
     channel = CLOG_CHANNEL_CLUSTER;
   }
 
+  string level = channels.get_log_level(channel);
+  int log_level = LogEntry::str_to_level(level);
+  if (!level.empty() &&
```

Comment:
What is `!level.empty()` condition for?  Wouldn't `log_level` be `CLOG_UNKNOWN = -1` in that case?

Reply:
The !level.empty() condition is for string empty check. The `get_str_map_key` can return empty string while fetching log level in string format for specific channel.